src/training/graph_group_sync.cpp

```diff
@@ -297,13 +297,26 @@ void SyncGraphGroup::update(std::vector<Ptr<data::Batch>> subBatches, size_t num
       return nullptr; // null if we reached beyond the end
   };
 
+  // Helper to quantize the model
+  auto quantizeModel = [&](size_t idx, size_t /*begin*/, size_t /*end*/) {
+    quantizers_[idx]->quantize(graphs_[idx]);
+  };
+
```

Comment:
I will likely break this with one of coming merges as I have rewritten the graph groups completely. Nothing to do here now, but I will ping you guys to help me get it right again.

Reply:
Same for the changes below.

Reply:
@afaji Could you provide a small working, reproducible example, which we can turn into a regression test? I think we have mentioned that already. This way we will minimize the risk of breaking this in future.

Reply:
@afaji FYI, tests can be added later, they are not a blocker for merging this PR. I created a task there: https://github.com/marian-nmt/marian-regression-tests/issues/63